src/main/java/ch/uzh/ifi/hase/soprafs23/entity/Ingredient.java

```diff
@@ -17,14 +19,18 @@ public class Ingredient implements Serializable  {
     private String name;
 
     @Column(nullable = true)
-    private Long calculatedRating;
+    private int calculatedRating;
 
     @ManyToMany(mappedBy = "ingredientsSet")
     private Set<User> usersSet;
 
     @ManyToOne
     private Group group;
 
+    @ElementCollection //TODO: makes sense to have own table?
```

Comment:
not sure tbh... the only need I can think of if after the final ingredients page is shown, if a user leaves then, then the ratings could be recalculated, but idk if that is something we want to do